policy_agent.py

```diff
@@ -106,6 +112,15 @@ def get_handler(self, resource_type, event_type):
         _log.debug("Looking up handler for event: %s", key)
         return self._handlers[key]
 
+    def load_cache(self):
+        """
+        Loads cache from etcd.
+        """
+        endpoints = self._client.get_endpoints()
```

Comment:
Actually scratch that: it won't do anything.  That means the place above where it is used is wrong (needs to specify hostname as well, or just don't bother).  pod.py:26 (maybe others).